### DISCUSSION TITLE
Where GDKX is ?

### DISCUSSION THREAD

**Discussion by Sentry-Thibault** [microsoft/GDK#67] [General]
Hi guys,

Can't find a link to download GDKX. I need it for Unity

Thanks

**Comment by danapayne** [comment]
The GDKX extension for Unity is currently only available via the Unity forums, in the private Xbox forum there. If you don't already have access to that forum, please contact your Microsoft Account Manager. If need further assistance, please reply here.

Thanks,
Dana
PM Middleware

**Comment by bbowman** [comment]
This discussion appears to be resolved now. Thank you @danapayne! Closing.
